libraries/botbuilder/src/teamsInfo.ts

```diff
@@ -333,6 +335,35 @@ export class TeamsInfo {
         return await this.getMemberInternal(this.getConnectorClient(context), t, userId);
     }
 
+    /**
+     * Sends a meeting notification to specific users in a Teams meeting.
+     *
+     * @param context The [TurnContext](xref:botbuilder-core.TurnContext) for this turn.
+     * @param notification The meeting notification payload.
+     * @param meetingId Id of the Teams meeting.
+     * @returns Promise with either an empty object if notifications were successfully sent to all recipients or
+     * [TeamsMeetingNotificationRecipientFailureInfos](xref:botframework-schema.TeamsMeetingNotificationRecipientFailureInfos) if notifications
+     * were sent to some but not all recipients.
+     */
+    static async sendMeetingNotification(
+        context: TurnContext,
+        notification: TeamsMeetingNotification,
```

Comment:
do we need null check for notification as well?

Reply:
we won't need a null check for `notification` because it is strongly typed and compile time enforced in Typescript. However despite it not being enforced in javascript, with IDEs like visual studio code, the type annotation is going to be visible.